### PR TITLE
add rule package_openldap-clients_removed

### LINKED ISSUE #5233: CIS 2.3.3 Ensure LDAP client is not installed (Scored)
Needs new rule.

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

add rule and include it into rhel8 cis

#### Rationale:

cis efford

- Fixes #5233

### CODE REVIEW

File: linux_os/guide/services/ldap/openldap_client/package_openldap-clients_removed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+documentation_complete: true
+
+title: 'Ensure LDAP client is not installed'
+
+description: |-
+    The Lightweight Directory Access Protocol (LDAP) is a service that provideso
```
Comment (jan-cerny) [inline linux_os/guide/services/ldap/openldap_client/package_openldap-clients_removed/rule.yml]:
```suggestion
    The Lightweight Directory Access Protocol (LDAP) is a service that provides
```

File: rhel8/profiles/cis.profile [hunk 2]
```diff
@@ -902,7 +902,7 @@ selections:
 
     ### 6.1.8 Ensure permissions on /etc/group- are configured (Scored)
 
-    
+
```
Comment (jan-cerny) [inline rhel8/profiles/cis.profile]: Nice catches these whitespace changes. But I usually recommend to not do them in unrelated commits. It might create conflicts or complicate backporting.

Comment (vojtapolasek) [reply rhel8/profiles/cis.profile]: Oh I did not notice that, I did not do it on purpose... probably my editor does that. Should I revert it somehow? What is actually being changed? It is hard to debug it for me here on GH.

File: linux_os/guide/services/ldap/openldap_client/package_openldap-clients_removed/rule.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: rhel7,ol7,rhel8,ol8,fedora,rhv4,ocp4
```
Comment (yuumasato) [inline linux_os/guide/services/ldap/openldap_client/package_openldap-clients_removed/rule.yml]: Nitpick: order the prodtypes alphabetically, this will reduce changes in the future.

File: rhel8/profiles/cis.profile (hunk 2 above)
Comment (yuumasato) [reply rhel8/profiles/cis.profile]: 42ee623912c55f895bb95fe2c23cf044ba89affa is the commit witch does unrelated changes.

**Comment by vojtapolasek** [comment]
It should be fixed now.

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by yuumasato** [comment]
@openscap-ci test this please
